editoast/src/models/fixtures.rs

```diff
@@ -101,6 +103,10 @@ pub fn simple_paced_train_base() -> PacedTrain {
             .expect("Unable to parse test train schedule");
     PacedTrain {
         train_schedule_base,
+        exceptions: vec![PacedTrainException {
+            id: Uuid::new_v4().to_string(),
```

Comment:
We don't need an uuid.

```suggestion
            id: "exception_id".into(),
```